### ISSUE TITLE
Page Not Found

### ISSUE THREAD

**Issue by ramondelafuente** [ansible/galaxy#1957]
##### SUMMARY
When clicking on any of my organisations role names to see the details I get redirected to https://galaxy.ansible.com/not-found.

This seems to be similar to #1907, but for namespace f500
Since that issue was closed, I did not want to comment there.

##### STEPS TO REPRODUCE
View any of the https://galaxy.ansible.com/f500 roles e.g.:

https://galaxy.ansible.com/f500/dumpall

##### EXPECTED RESULTS
The page displaying details about the role.

##### ACTUAL RESULTS
Redirected to https://galaxy.ansible.com/not-found (Page Not Found).

**Comment by ironfroggy** [comment]
There seem to be duplicate namespaces in galaxy, spelled "f500" and "F500".

This namespace seems to be unused, but causing the conflicts due to its non-lowercase name form: https://galaxy.ansible.com/api/v1/namespaces/5884/

**Comment by ramondelafuente** [comment]
Yeah, I figured that based on the previous error - but I have no idea how (if even) I caused this, and if I can fix this. I don't see the F500 namespace in my interface.

**Comment by ramondelafuente** [comment]
Interesting. I did not remember this as being an open issue (since this problem had gone away on it's own). But now it has returned, I can't get to any of our roles in the galaxy web-interface.

See https://github.com/ansible/galaxy/issues/2237

**Comment by ramondelafuente** [comment]
The roles are now displying again in galaxy.
